packages/apsara-ui/src/Skeleton/Skeleton.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { ListSkeleton, EditorSkeleton, DetailsSkeleton } from "./Skeleton";
+
+export default {
+    title: "Feedback/Skeleton",
```

Comment:
should we rename this to `Loading` instead of `Skeleton`? Seems more intuitive, or is `Skeleton` a common term?

Reply:
I suppose `Loading/Loader` alone is rather confusing, CMIIW it gives sense of that circular loading animation we have.
`Skeleton` is rather a common term for this one and I think it is being used in many other common UI libraries.
@StewartJingga maybe we can rename it to `SkeletonLoading` or `SkeletonLoader`.
@ravisuhag @AkarshSatija What do you guys suggest for naming this one?

Reply:
just did a quick research, and Skeleton seems to be pretty common. We can keep it.